src/adagrams.js

```diff
@@ -1,8 +1,160 @@
-const Adagrams = {
-  drawLetters() {
-    // Implement this method for wave 1
-  },
-};
+class Adagrams {
+  static drawLetters() {
+
+    let letters = [
+      'A','A','A','A','A','A','A','A','A',
+      'B','B',
+      'C','C',
+      'D','D','D','D',
+      'E','E','E','E','E','E','E','E','E','E','E','E',
+      'F','F',
+      'G','G','G',
+      'H','H',
+      'I','I','I','I','I','I','I','I','I',
+      'J','J',
+      'K',
+      'L','L','L','L',
+      'M','M',
+      'N','N','N','N','N','N',
+      'O','O','O','O','O','O','O','O',
+      'P','P',
+      'Q',
+      'R','R','R','R','R','R',
+      'S','S','S','S',
+      'T','T','T','T','T','T',
+      'U','U','U','U',
+      'V','V',
+      'W','W',
+      'X',
+      'Y','Y',
+      'Z',
+    ];
+
+    let number = letters.length;
+    
+    function getRandomInt(number) {
```

Comment:
If you're going to have an inner function, I would appreciate it if you used functional expression syntax, so `const getRandomInt = function getRandomInt(number) {`